src/roiextractors/extractors/tiffimagingextractors/scanimagetiffimagingextractor.py

```diff
@@ -22,13 +24,487 @@ def _get_scanimage_reader() -> type:
     ).ScanImageTiffReader
 
 
-class ScanImageTiffImagingExtractor(ImagingExtractor):
+def extract_extra_metadata(
```

Comment:
It's a personal opinion, but I'm usually of the view that for code suites like this, it is cleaner to move these types of helper functions (particularly for metadata parsing) to an adjacent utils file such as `scanimage_utils.py` to free up visual space in the main file for the primary extractors

What do you think about that approach?

Reply:
Sure! That seems reasonable.  Maybe we could put them in the `extraction_tools` module and rename them to be specific to scanimage i.e. `extract_scanimage_metadata` and `parse_scanimage_metadata`?

Reply:
I would still keep them scoped locally to the `extractors` subfolder (though I'd also advocate for this to be in a further subfolder `scanimage` distinct from generic `tiff` reader, but that's maybe beyond this PR) to make it obvious they are not to be used more generally

Reply:
> I would still keep them scoped locally to the extractors subfolder

Ok!

> I'd also advocate for this to be in a further subfolder scanimage distinct from generic tiff reader

What do you think about flattening the tiff extractors in general?  They don't actually share any common utils, so maybe we should just have separate folders for scanimage, bruker, tiff, and micromanager?

Reply:
What do you think about flattening the tiff extractors in general?

Sounds great


> They don't actually share any common utils, so maybe we should just have separate folders for scanimage, bruker, tiff, and micromanager?

Exactly, that's how we treat them in NeuroConv anyway, IDK why it never happened here

BTW this can be a follow-up PR

Reply:
See #246